frontend/src/components/Dropdown/MultiSelect/MultiSelectProvider.tsx

```diff
@@ -84,6 +85,7 @@ export const MultiSelectProvider = ({
   downshiftComboboxProps = {},
   downshiftMultiSelectProps = {},
   inputAria,
+  overrideVirtualListItemHeight, // NOTE: for use when virtual list item height is not VIRTUAL_LIST_ITEM_HEIGHT e.g, when adding description line
```

Comment:
Typing issue here? Don't see this declared in `MultiSelectProviderProps`

Reply:
Added to `MultiSelectProviderProps`. Chose to keep this in so that we can override in the future since the label description has been added.